templates/mapfishapp/mapfishapp-deployment.yaml

```diff
@@ -65,4 +65,4 @@ spec:
       imagePullSecrets:
       - name: {{ $webapp.registry_secret }}
       {{- end }}
-{{- end }}
```

Comment:
could you avoid removing the return line at the end of the file? there are other files where you removed the end of file.

Reply:
indeed. I didn't do it on purpose. I'll fix this

Reply:
What I don't get is that on my code editor, I can still see the new lines. It looks like git is doing it to me on commit